packages/core/src/file/file.element.ts

```diff
@@ -64,15 +64,21 @@ export class CdsFile extends CdsControl {
 
   firstUpdated(props: Map<string, any>) {
     super.firstUpdated(props);
-    this.inputControl.addEventListener('change', e => this.updateLabel((e.target as any).files));
+    this.inputControl.addEventListener('change', e => this.updateLabelAndFocus((e.target as any).files));
   }
 
-  private updateLabel(files?: FileList) {
+  private updateLabelAndFocus(files?: FileList) {
     if (files && files.length) {
       this.buttonLabel = files.length > 1 ? `${files.length} ${CommonStringsService.keys.files}` : files[0].name;
     } else {
       this.buttonLabel = CommonStringsService.keys.browse;
       this.inputControl.value = '';
+
+      const browseButton = this.shadowRoot?.querySelector('cds-button');
```

Comment:
What do you think about setting a `const enum` with our root element selectors e.g. `cds-button`, `cds-control`, `cds-icon`, etc.? I think it will be helpful to change the selector in the enum instead of search/replace. Some possible situation for this is for example changing the prefix of the library.

Reply:
Not a bad idea and we can consider it in a separate PR! This PR contains unit tests that would fail if the prefix changed so it wouldn't go unnoticed.